MovieSharing/VideoRequest.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  VideoRequest.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 29.09.21.
+//
+
+import Foundation
+
+enum VideoError:Error{
+    case noDataAvailable
+    case canNotProcessData
+    
+}
+
+struct VideoRequest {
+    let resourceURL: URL
+    
+    init(){
```

Comment:
please use a single space between the parameter parenthesis and the method implementation curly brackets